pkgs/misc/tmux-plugins/default.nix

```diff
@@ -92,12 +92,12 @@ in rec {
 
   catppuccin = mkTmuxPlugin {
     pluginName = "catppuccin";
-    version = "unstable-2023-07-15";
+    version = "unstable-2023-08-21";
     src = fetchFromGitHub {
       owner = "catppuccin";
       repo = "tmux";
-      rev = "e7b50832f9bc59b0b5ef5316ba2cd6f61e4e22fc";
-      hash = "sha256-9ZfUqEKEexSh06QyR5C+tYd4tNfBi3PsA+STzUv4+/s=";
+      rev = "7a284c98e5df4cc84a1a45ad633916f0b2b916b2";
+      sha256 = "1w2vccjxsm8h82299w6vl4lqvlni5i1dsk65ny97cz8491dk25wg";
```

Comment:
```suggestion
      hash = "sha256-jxcxW0gEfXaSt8VM3UIs0dKNKaHb8JSEQBBV3SVjW/A=";
```

Reply:
I will accept your change but just for my own education, @SuperSandro2000 how did you get this hash and what's the difference?

Reply:
Using hash is the future because it makes using other hash algorithms easy and does not require code changes, just nix needs to understand it. This way overlay which change hashes also can't break which one of the main reasons why we should not change this back and forth. 

I got the hash by running rnix-hashes

```
 ▶ rnix-hashes 1w2vccjxsm8h82299w6vl4lqvlni5i1dsk65ny97cz8491dk25wg
SRI     sha256-jxcxW0gEfXaSt8VM3UIs0dKNKaHb8JSEQBBV3SVjW/A=
base16  8f17315b48047d7692b7c54cdd422cd1d28d29a1dbf09484401055dd25635bf0
base32  1w2vccjxsm8h82299w6vl4lqvlni5i1dsk65ny97cz8491dk25wg
base64  jxcxW0gEfXaSt8VM3UIs0dKNKaHb8JSEQBBV3SVjW/A=
```

but nix can also convert it but that is a little bit more verbose and does not guess the type

```
 ▶ nix hash to-sri sha256:1w2vccjxsm8h82299w6vl4lqvlni5i1dsk65ny97cz8491dk25wg
sha256-jxcxW0gEfXaSt8VM3UIs0dKNKaHb8JSEQBBV3SVjW/A=
```